lib/tlaw/dsl/base_wrapper.rb

```diff
@@ -0,0 +1,48 @@
+require_relative 'post_process_proxy'
+
+module TLAW
+  module DSL
+    class BaseWrapper
+      def initialize(object)
+        @object = object
+      end
+
+      def define(&block)
+        instance_eval(&block)
+      end
+
+      def description(text)
+        # first, remove spaces at a beginning of each line
+        # then, remove empty lines before and after docs block
+        @object.description =
+          text
+          .gsub(/^[ \t]+/, '')
+          .gsub(/\A\n|\n\s*\Z/, '')
+      end
```

Comment:
The last thing that I noticed: let's probably remove this logic completely? It just repeats what Ruby 2.3 "squiggly heredoc" do, and as this PR specifies 2.3 as the minimum version, there is no need to replicating language features. One can just
```ruby
  description <<~D
    Any description
    goes here.

    Like a boss!
  D
```
...and be happy.